### ISSUE TITLE
Cook books

### ISSUE THREAD

**Issue by giulioungaretti** [QCoDeS/Qcodes#453]
CookBooks should address the things more generally.
These were requested a year ago but like many other things ignored (#124).

- [ ] How to use the station
- [ ] How to use an instrument
- [ ] How to use parameters
- [ ] How to organize an experiment
- [ ] How to run a loop
- [ ] How to plot live data
- [ ] How to save the data
- [ ] How to read the data back, and plot it in the first place
- [ ] How to use logging and user-logging (for experiment logs)
- [ ] How to save metadata and logs

**Comment by WilliamHPNielsen** [comment]
I'd certainly like to chip in on this.

**Comment by peendebak** [comment]
@WilliamHPNielsen @unga Are the cookbooks like the examples in https://github.com/QCoDeS/Qcodes/tree/master/docs/examples?

I think if would help to add a `README.md` in the `qcodes/docs/examples` directory with a list of the most important examples (`Tutorial.ipynb` would be on top of the list). Also I think the `Tutorial.ipynb` should be simplified and some examples in that notebook moved to other examples.

**Comment by giulioungaretti** [comment]
@peendebak  yes very similar but smaller, so that it's easier to digest them (or even copy paste).
Agree with the tutorial, it was never written for somebody that is really a beginner, as it has been show in qdev  💥💥
